client/containers/DashboardReviews/DashboardReviews.tsx

```diff
@@ -50,6 +57,9 @@ const DashboardReviews = (props: Props) => {
 							{pub.reviews
 								.sort((a, b) => Date.parse(b.createdAt) - Date.parse(a.createdAt))
 								.map((review) => {
+									const hasReviewers =
+										review.reviewer && review.reviewer.length > 0;
+									const reviewers = review.reviewer;
```

Comment:
i want to wait on that. it would be nice to do exactly that but we gotta think about how a user can be represented as an attribution. im thinking as a MinimulUser or User. Instead of attributable user for pub or collection and reviewer. for now im fine with it